src/android/app/src/main/jni/game_metadata.cpp

```diff
@@ -71,6 +76,32 @@ RomMetadata GetRomMetadata(const std::string& path) {
 
 extern "C" {
 
+jboolean Java_org_yuzu_yuzu_1emu_utils_GameMetadata_getIsValid(JNIEnv* env, jobject obj,
+                                                               jstring jpath) {
+    const auto file = EmulationSession::GetInstance().System().GetFilesystem()->OpenFile(
+        GetJString(env, jpath), FileSys::Mode::Read);
+    if (!file) {
+        return false;
+    }
+
+    auto loader = Loader::GetLoader(EmulationSession::GetInstance().System(), file);
+    if (!loader) {
+        return false;
+    }
+
+    const auto fileType = loader->GetFileType();
```

Comment:
```suggestion
    const auto file_type = loader->GetFileType();
```
we use underscores and lowercase in c++ variable names